lldb/source/API/SBThread.cpp

```diff
@@ -114,14 +114,15 @@ SBThread::operator bool() const {
   LLDB_INSTRUMENT_VA(this);
 
   std::unique_lock<std::recursive_mutex> lock;
```

Comment:
Again, I'm not sure why we decided to do all this work without first checking that m_opaque_sp is non-null.